src/dealDashboard/dealVotes/dealVotes.ts

```diff
@@ -47,24 +47,19 @@ export class DealVotes {
       voteText: "Voting is closed. You can find the voting history below.",
       statusText: "Token Swap Failed",
     },
-    {
-      condition: () => !this.deal.majorityHasVoted && !this.deal.isAuthenticatedRepresentativeUser && !this.deal.isAuthenticatedProposalLead,
-      voteText: "Waiting for the representatives to vote",
-      statusText: "Voting is progress",
-    },
     {
       condition: () => !this.deal.majorityHasVoted && this.deal.isAuthenticatedRepresentativeUser && this.deal.hasRepresentativeVoted && !this.deal.isAuthenticatedProposalLead,
       voteText: "You have cast your vote. Please wait for other representatives to cast theirs. You are able to change your vote before the funding phase is initiated. Once the deal is approved, the Proposal Lead will initiate the funding phase",
       statusText: "Voting in progress",
     },
     {
-      condition: () => !this.deal.majorityHasVoted && this.deal.isAuthenticatedRepresentativeUser && !this.deal.hasRepresentativeVoted && !this.deal.isAuthenticatedProposalLead,
-      voteText: "The deal will be approved once the majority has voted in favor. Once the deal is approved, the Proposal Lead will initiate the funding phase",
+      condition: () => !this.deal.majorityHasVoted && this.deal.isAuthenticatedProposalLead,
+      voteText: "The deal will be approved once the majority has voted in favor. Once the deal is approved, you can initiate the funding phase",
       statusText: "Voting in progress",
     },
     {
-      condition: () => !this.deal.majorityHasVoted && this.deal.isAuthenticatedProposalLead,
-      voteText: "The deal will be approved once the majority has voted in favor. Once the deal is approved, you can initiate the funding phase",
+      condition: () => !this.deal.majorityHasVoted,
+      voteText: "The deal will be approved once the majority has voted in favor. Once the deal is approved, the Proposal Lead will initiate the funding phase",
```

Comment:
nitpick:

If you would have changed the order of the right side, it would have only showed as one diff

![image](https://user-images.githubusercontent.com/30693990/171044603-cc601384-c15c-4380-ad5d-1b2c444d12a5.png)